.github/workflows/drools-lsp-pr.yml

```diff
@@ -41,3 +41,26 @@ jobs:
       - uses: actions/checkout@v3
       - name: Build with Maven
         run: mvn -B clean install
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - run: npm install
+        working-directory: ./client
+
+      - name: Extension E2E Test for Linux
+        run: xvfb-run -a npm test
+        working-directory: ./client
+        if: runner.os == 'Linux'
+
+      - name: Extension E2E Test for Windows
+        run: npm test
+        working-directory: ./client
+        if: runner.os == 'Windows'
+
+      - name: Extension E2E Test for MacOS
+        run: GHA_JAVA_HOME=$JAVA_HOME npm test
+        working-directory: ./client
+        if: runner.os == 'macOS'
```

Comment:
This `GHA_JAVA_HOME` is a little tricky. There is an issue that `JAVA_HOME` is changed to jdk8 during `npm test` only in Github Action + MacOS. I'll investigate the issue later, but I hope this workaround is acceptable.